code_challenges/code_wars/pig_latin.py

```diff
@@ -0,0 +1,18 @@
+def pig_it(text):
+    text = text.split(" ")
+    translated_text = ""
+    punctuations = [" ", ".", "!", "?"]
```

Comment:
This should be a set, tell my why?

Reply:
theres only one of each option and it is quicker

Reply:
Why is it quicker?

Reply:
the run time is O(n) vs O(1). With the list it has to loop through the list every time. With the set it only needs to do it once then that information is stored

Reply:
> With the set it only needs to do it once then that information is stored

This isn't correct.  See if you can good why a set is O(n) lookup.